### PR TITLE
Reads active node info, sets as default values for device config

### LINKED ISSUE #172: Set defaults in settings UI
Read in a node's current name, and have that set as the default in the settings UI instead of having a placeholder. 

This issue also doubles as a learning ticket for react redux.

### PR DESCRIPTION
**Pull request by sherry1iu**
PR adds in code that reads the active node's information to populate default imputs in device config. 

settings init

Also changed appearance of alerts:
image

Closes #172

### CODE REVIEW

File: src/components/Settings/Settings.tsx [hunk 1]
```diff
@@ -84,6 +99,7 @@ const Settings = () => {
                 minLength={1}
                 value={deviceNickname}
                 onChange={(e) => setDeviceNickname(e.target.value)}
+                disabled={selectActiveNode === null ? true : false}
```
Comment (ajmcquilkin) [inline src/components/Settings/Settings.tsx:102]: Nit: you don't need a ternary here, the condition already evaluates to true or false

File: src/components/Settings/Settings.tsx [hunk 2]
```diff
@@ -68,6 +82,7 @@ const Settings = () => {
                 label="Personalised name for this device."
                 value={deviceName}
                 onChange={(e) => setDeviceName(e.target.value)}
+                disabled={selectActiveNode == null ? true : false}
```
Comment (ajmcquilkin) [inline src/components/Settings/Settings.tsx:85]: Nit: same here

**Review by ajmcquilkin — APPROVED** [review]

File: src/components/Settings/Settings.tsx (hunk 1 above)
Comment (sherry1iu) [reply src/components/Settings/Settings.tsx:102]: Added it because I got tired of seeing the red lines lol

Comment (ajmcquilkin) [reply src/components/Settings/Settings.tsx:102]: What red lines were you seeing? That should evaluate to a boolean expression

Comment (sherry1iu) [reply src/components/Settings/Settings.tsx:102]: Just realized I was evaluating the function instead of calling it; will make new commit to fix this

Comment (ajmcquilkin) [reply src/components/Settings/Settings.tsx:102]: Oh yeah, note that you shouldn't evaluate selectors inline. I would call it with the `useSelector` hook outside of the JSX then use that resulting value

Comment (ajmcquilkin) [reply src/components/Settings/Settings.tsx:102]: Looks like you could use the `activeNode` variable here

Comment (sherry1iu) [reply src/components/Settings/Settings.tsx]: That works, thanks!

File: src/components/Settings/Settings.tsx [hunk 3]
```diff
@@ -84,6 +99,7 @@ const Settings = () => {
                 minLength={1}
                 value={deviceNickname}
                 onChange={(e) => setDeviceNickname(e.target.value)}
+                disabled={activeNode ? false : true}
```
Comment (ajmcquilkin) [inline src/components/Settings/Settings.tsx:102]: Nit: could also use `!activeNode` which converts to `true` only if `activeNode` is `null`

**Comment by ajmcquilkin** [comment]
Should be good to merge! Lmk and I can approve once the PR is marked as ready for review.

**Comment by sherry1iu** [comment]
I want to play with it a little more, but thanks!

**Review by ajmcquilkin — APPROVED** [review]

**Pull request merged**